dev-lang/ispc/ispc-9999.ebuild

```diff
@@ -36,10 +36,12 @@ BDEPEND="
 
 PATCHES=(
 	"${FILESDIR}/${PN}-1.13.0-cmake-gentoo-release.patch"
-	"${FILESDIR}/${PN}-1.14.0-llvm-10.patch"
-	"${FILESDIR}/${PN}-1.13.0-werror.patch"
+	"${FILESDIR}/${PN}-9999-llvm.patch"
+	"${FILESDIR}/${PN}-9999-werror.patch"
 )
 
+CMAKE_BUILD_TYPE="Release"
```

Comment:
Why is this needed?

Reply:
A check has been added so trying to use anything but the standard build types will make the cmake scripts error out.

Reply:
Use `RelWithDebInfo`, that's the one that is least likely to break stuff.